### ISSUE TITLE
A11y_SQLAzureDataStudio_AddAzureAccount_AddAzureAccount_ScreenReader:The tree view is announced as table

### ISSUE THREAD

**Issue by BhanuPrakashGit** [microsoft/azuredatastudio#22493]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS1.3.1;#VoiceOver;#SQLADS-MAC-Feb2020;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.43.0-insider (Universal)
Commit: d3d594e8267f536c481bb3e4cdfefde56ddfdb59
Date: 2023-03-09T18:15:11.027Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.4.0

### Additional Details:
Mac OS: Ventura 13.3
Screen reader: Voice Over.
MAS References: MAS1.3.1

### Repro Steps:

Launch Azure Data Studio application.
Connect to server with windows credentials.
Navigate to Connections from left menu.
Select Azure, navigate to + button & hit enter.
Navigate to Add account button in right side window & hit enter.
Launch screen reader as Voice over
### Actual:
On navigating to the tree view items "Steffi " it is announced as table.
Each item is announced as list box.
There is not level announced for the items.

### Expected:
On navigating to the tree view items "Steffi " it is not a table so the word table should not be announced.
Each item should be announced as a button
There should be proper hierarchy levels for the items.

### User Impact:
Screen reader users will receive the wrong information of the structure as well as the control and hierarchy will be conveyed in a wrong manner.

### Attachment link for Reference:

![74931849-82402400-5406-11ea-9958-700f13dc2fa2](https://user-images.githubusercontent.com/104351566/228226146-cc4d9fae-5798-4388-a065-7c129c8a21e4.png)
![74931818-6fc5ea80-5406-11ea-957f-d07ca1db08d7](https://user-images.githubusercontent.com/104351566/228226159-01033928-ddbf-408c-a94d-ff8871f8ed2b.png)
![74931784-5cb31a80-5406-11ea-848f-924e46ffdeda](https://user-images.githubusercontent.com/104351566/228226161-d26959fd-dd4b-4845-a4c9-a8226aac3a4c.png)

**Comment by BhanuPrakashGit** [comment]
Reference bug: https://github.com/microsoft/azuredatastudio/issues/9234

**Comment by cssuh** [comment]
@BhanuPrakashGit do you know if this bug exists in the intel Mac version of ADS? I only ask because I do not have access to an ARM64 device for debugging.

**Comment by cssuh** [comment]
Hi @BhanuPrakashGit, https://github.com/cssuh/treeViewExample/blob/main/test1.html I have made a simple example tree view where the voiceover on mac will read it as a table, even though its role is a treeview.  It can also be seen here: https://microsoftedge.github.io/DevToolsSamples/bugs/list-as-table.html and in the [W3 example for treeview](https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-1b/) where voiceover on mac will announce the treeview as a table.

This only happens in voiceover on mac, as these treeviews do not get identified as a table in Windows.

**Comment by BhanuPrakashGit** [comment]
Logged External IRT bug.

**Comment by cssuh** [comment]
Closing as this is an external bug
